packages/atomic/src/components/common/query-error/link.tsx

```diff
@@ -0,0 +1,34 @@
+import {FunctionalComponent, h} from '@stencil/core';
+import {i18n} from 'i18next';
+import {KnownErrorType} from './known-error-types';
+
+interface QueryErrorLinkProps {
+  errorType?: string;
+  i18n: i18n;
+}
+
+export const QueryErrorLink: FunctionalComponent<QueryErrorLinkProps> = ({
+  errorType,
+  i18n,
+}) => {
+  const getErrorLink = () => {
+    switch (errorType as KnownErrorType) {
+      case 'NoEndpointsException':
+        return 'https://docs.coveo.com/en/mcc80216';
+      case 'InvalidTokenException':
+        return 'https://docs.coveo.com/en/102';
+      case 'OrganizationIsPausedException':
+        return 'https://docs.coveo.com/l6af0467';
+      default:
+        return null;
+    }
+  };
```

Comment:
Small detail, This function will be recreated each time the component renders, I personally would declare it outside the component with the errorType as a param or wrap it in a memo with the errorType as a dependency.